alicloud/resource_alicloud_oss_bucket_test.go

```diff
@@ -331,6 +331,38 @@ func TestAccAlicloudOssBucketStorageClass(t *testing.T) {
 	})
 }
 
+func TestAccAlicloudOssBucketSseRule(t *testing.T) {
+	var bucket oss.BucketInfo
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+
+		// module name
+		IDRefreshName: "alicloud_oss_bucket.sserule",
+		Providers:     testAccProviders,
+		CheckDestroy:  testAccCheckOssBucketDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAlicloudOssBucketSseRuleConfig(acctest.RandInt()),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckOssBucketExists(
+						"alicloud_oss_bucket.sserule", &bucket),
+					resource.TestCheckResourceAttr(
+						"alicloud_oss_bucket.sserule",
+						"server_side_encryption_rule.0.sse_algorithm",
+						"KMS"),
+					resource.TestCheckResourceAttr(
+						"alicloud_oss_bucket.sserule",
+						"server_side_encryption_rule.0.kms_master_key_id",
+						"11233455"),
+				),
+			},
+		},
+	})
+}
```

Comment:
Please add a step to test updating server_side_encryption_rule.

Reply:
done